bin/pycbc_live

```diff
@@ -359,6 +361,55 @@ class LiveEventManager(object):
                 cmd, shell=True, stdout=logfile, stderr=logfile
             )
 
+    def create_gdb(self):
+        """
+        Function to create GraceDB session.
+        """
+        from ligo.gracedb.rest import GraceDb
+        logging.info('Creating GraceDB session')
+        # Set up dict for args that reload the certificate if it will expire in the reload_buffer time
+        # These args are documented here: https://ligo-gracedb.readthedocs.io/en/latest/api.html#ligo.gracedb.rest.GraceDb
+        # Because we do not change any of the request session values when running the code, it should remain thread safe. 
+        gdbargs = {'reload_certificate': True, 'reload_buffer': 300}
+        self.gracedb = GraceDb(args.gracedb_server, **gdbargs) if args.gracedb_server else GraceDb(**gdbargs)
```

Comment:
Avoid using `args` here, as it is not an argument of the method. I think `gracedb_server` is one of the attributes of `self`.

Reply:
You can also simplify this line by (optionally) putting the server URL in `gdbargs` instead.

Reply:
I definitely agree, and I was originally going to assign them during the `__init__` of the class e.g. `self.gracedb_server = args.gracedb_server`, however, I did notice that `args` is used in a bunch of other class methods already without being explicitly passed in to those either? I'm unsure which convention we want to adopt?

Reply:
That is right, a lot of the code does pass around `args`, however I would like *new* code to stop doing that.

Reply:
Yes I agree. Would it be ok to leave this as it is, then I can do a separate branch fixing it for all of the methods in that class?

Reply:
Fair enough.